src/menu/menu.scss

```diff
@@ -149,6 +150,32 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
       }
     }
   }
+
+  &-skeleton {
+    --iui-menu-item-skeleton-max-width: 30%;
+    cursor: auto;
+
+    .iui-content {
+      max-width: var(--iui-menu-item-skeleton-max-width);
```

Comment:
> this is kinda misleading as it doesnt include the icon in this max-width

Is it really that big of a problem that icon size is not added? Because it is used just to mimic content.

>I assume skeleton menu items wouldn't be selectable, but if they are then the max-width seems to cause the checkmark icon to be placed incorrectly.

Is there a case where selected checkmark should be shown in skeleton? Because then I could work to fix it.

Reply:
I just meant the name is misleading, maybe add `content` in it. Of course ideally it would be nice if the icon could be somehow included in the max-width, but that might be difficult

Reply:
Renamed.